src/cart/slice/slice.ts

```diff
@@ -34,5 +51,24 @@ const slice = createSlice({
 
 export const selectCartSlice = (state: RootState) => state.cart;
 
+export const selectProductsFromCart = createSelector(
+  [selectCartSlice, selectIProductsSlice],
```

Comment:
Why the "I" in the selector name, moreover a slice should not be aware of another slice structure, so we should use a more specific selector, not the slice but the products list